julearn/model_selection/__init__.py

```diff
@@ -0,0 +1,3 @@
+from . cv import StratifiedBootstrap
```

Comment:
Missing
```Python
# Authors: Sami Hamdan <s.hamdan@fz-juelich.de>
# License: AGPL
```

Reply:
Added both of us: as we did for all the other files in julearn (except for the examples)